react-client/src/Components/App.js

```diff
@@ -46,13 +46,20 @@ function App() {
       if (e.target.id !== "closedompeeker" && domSwitch == true) {
         e.preventDefault();
 
+        const clsArr =[...e.target.classList].map((cls) => {
```

Comment:
You can do this instead: 
```suggestion
        [...e.target.classList].map((cls) => ({ clsName: `.${cls}` });
```